projects/knora-ui/src/lib/viewer/values/boolean-value/boolean-value.component.html

```diff
@@ -11,7 +11,6 @@
             <textarea
                     matInput
                     cdkTextareaAutosize
```

Comment:
`cdkTextareaAutosize `, `cdkAutosizeMinRows`, and `cdkAutosizeMaxRows` are still needed in the template?

Reply:
They're still needed in the template so that the user can see up to six lines before it will start to scroll. Otherwise the user will only have one line with a scrollbar which is not very user friendly.

Reply:
ok, got it!